src/User32.Desktop/User32+PrintWindowFlags.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="PrintWindow"/> nested enum.
```

Comment:
We tend to use `nested type.` instead of specifying the type of what is nested in recent files.

But I just noticed that there are still a lot of the old versions so don't consider that as problematic, i'll create an issue to track fixing that everywhere.